### DISCUSSION TITLE
Introduce Yourself

### DISCUSSION THREAD

**Discussion by sofia-bg** [UNC-Project-Open-AAC/OS-DPI#82] [General]
Please use this thread to introduce yourself to the community. Include your name, and why you are interested in the Project Open Design Community.

**Comment by sofia-bg** [comment]
Hello! I am Sofia Benson-Goldberg. I am a postdoctoral research associate at the Center for Literacy and Disability Studies. I am a researcher on Project Open. I am excited to use the designer interface to come up with novel solutions that support AAC technology users in improving their face to face communication experiences.

**Comment by frrancode** [comment]
Hello everyone! My name is Franchesca Arecy and I am a doctoral student in the Communicative and Assistive Device Lab at University at Buffalo! I am interested the development of assistive technology. Looking forward to "meeting" you all :)!

**Comment by kaericks** [comment]
Hello, I am Karen Erickson. I am the Director of the [Center for Literacy & Disability Studies](https://www.med.unc.edu/ahs/clds) at UNC. I am also the PI on Project Open. Thank you for your interest in this community.

**Comment by lageist** [comment]
Hi! I'm Lori Geist, an SLP and member of the Project Open team at the Center for Literacy and Disability Studies at UNC at Chapel Hill. Looking forward to the exchange of AAC design ideas here.

**Comment by dtnader** [comment]
Hi everyone, I'm Danielle Nader. I am a doctoral student in Speech & Hearing Sciences at UNC Chapel Hill and I work as a research assistant in the Center for Literacy and Disability Studies.  I am interested in learning more about features of design that support meaningful communication for both aided speakers and their partners.

**Comment by gbishop** [comment]
Hi. I'm Gary Bishop, a retired professor of Computer Science at UNC. I've been working with CLDS for years on projects related to disability and literacy. Programming is my favorite activity.

**Comment by kconway5** [comment]
Hi all, I’m Kayla Conway. I am a doctoral student in Communicative Disorders and Science and work in the Communicative and Assistive Device Lab (CADL) at University at Buffalo. I’m looking forward to learning more about the design development side of AAC.

**Comment by asatchidanand** [comment]
Hi everyone!  I'm Antara Satchidanand and I am a PhD student at the University at Buffalo.  I come to Communication Disorders from teaching middle school language arts and literacy.  I am really interested in how people interact and how technology affects this.  I am excited to be part of a project that looks at how AAC can be developed to help people participate in conversation the way they would like to.
